mirascope/core/_internal/utils.py

```diff
@@ -50,6 +52,15 @@ def format_prompt_template(template: str, attrs: dict[str, Any]) -> str:
                 values[var] = "\n".join([str(item) for item in attr])
         else:
             values[var] = str(attr)
+    return values
+
+
+def format_prompt_template(template: str, attrs: dict[str, Any]) -> str:
```

Comment:
maybe we should change this name to just `format_template`

Reply:
Nice catch. Completely forgot to change it later.